source/tools/testConsistency/android-renderer/AndroidDriver/build.gradle

```diff
@@ -0,0 +1,24 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        google()
+        jcenter()

```

Comment:
@golddove / @almedina-ms could you double check the gradle files here to make sure these match the new format you guys are trying to move to and/or remove any redundancies? thanks